mock/datasets/nighttime-lights.data.mdx

```diff
@@ -12,6 +12,16 @@ media:
 thematics:
   - covid-19
   - agriculture
```

Comment:
`thematics` can probably be removed now?

Reply:
yes. this will need to move under `taxonomy.Topics`